cli/cmd/aeon.go

```diff
@@ -24,13 +27,17 @@ var connectCtx = aeoncmd.ConnectCtx{
 	Transport: aeoncmd.TransportPlain,
 }
 
+var configPath string
+var instance string
+
 func newAeonConnectCmd() *cobra.Command {
 	var aeonCmd = &cobra.Command{
 		Use:   "connect URI",
 		Short: "Connect to the aeon instance",
 		Long: `Connect to the aeon instance.
 tt aeon connect localhost:50051
-tt aeon connect unix://<socket-path>`,
+tt aeon connect unix://<socket-path>
+tt aeon connect -c path instanceName>`,
```

Comment:
```suggestion
tt aeon connect -c path INSTANCE_NAME`,
```
To correlate with `tt connect/play` help style.